backend/s390x.pm

```diff
@@ -69,15 +66,15 @@ sub do_stop_vm {
 
     # now cleanly disconnect from the guest and then kill the local
     # Xvnc
-    $self->deactivate_console({testapi_console => 'sut'});
-    $self->deactivate_console({testapi_console => 'worker'});
+    # FIXME: "console worker does not exist at /home/os-autoinst/consoles/s3270.pm line 491 thread 1."
```

Comment:
that's very old code from froh. Just erase it. we can revive it if we found a use